server/core/runtime/state_rm_step_runner_test.go

```diff
@@ -0,0 +1,61 @@
+package runtime
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	. "github.com/petergtz/pegomock"
+	"github.com/runatlantis/atlantis/server/core/runtime/mocks/matchers"
+	"github.com/runatlantis/atlantis/server/core/terraform/mocks"
+	matchers2 "github.com/runatlantis/atlantis/server/core/terraform/mocks/matchers"
```

Comment:
Can we call this `tfmatchers` and call the other `matchers` => `runtimematchers` or something?

```suggestion
	tfmatchers "github.com/runatlantis/atlantis/server/core/terraform/mocks/matchers"
```